SPCTR-3234 Improvement: Forms - If the  field is empty in form submissions, the default recipient will be the current user's WordPress email address. (#3503)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Image Gallery - Resolved an issue with the masonry type wherein the loader would not be hidden after all the images have been loaded.
* Fix: Global - Resolved an issue in which the font-family that was set would not appear as intended in the FSE template editor.
* Fix: Separator - Resolved an issue wherein the height would revert from percentage to pixels after reloading the editor.

> * Improvement: Forms - If the `To` field is empty in form submissions, the default recipient will be the current user's WordPress email address.

Reviewer: Can we move this improvement up with the others?

```suggestion
* Improvement: Form - Made the default recipient as the current user's WordPress email address when the `To` field is left empty in submitted forms.
```

Author: Done
